test/e2e/standard/osbachecker.go

```diff
@@ -0,0 +1,86 @@
+package standard
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+
+	"github.com/openshift/openshift-azure/pkg/util/random"
+)
+
+type osbaObj struct {
+	name string
+	data []byte
+}
+
+var osbaApp, osbaService osbaObj
+
+func init() {
+	osbaApp.name = "osba-app"
+	osbaService.name = "osba-service"
+
+	var err error
+	osbaService.data, err = ioutil.ReadFile("../manifests/osba/osba-os-template.yaml") // running via `go test`
+	if os.IsNotExist(err) {
+		osbaService.data, err = ioutil.ReadFile("test/e2e/manifests/osba/osba-os-template.yaml") // running via compiled test binary
+	}
+	if err != nil {
+		panic(err)
+	}
+}
+
+// ValidateOSBAApp verifies that the created application runs normally
+func (sc *SanityChecker) ValidateOSBAApp(ctx context.Context, cookie interface{}) []*TestError {
+	return nil
+}
+
+// DeleteOSBAApp deletes the OSBA Application after tests completed
+func (sc *SanityChecker) DeleteOSBAApp(ctx context.Context, cookie interface{}) []*TestError {
+	var errs []*TestError
+	sc.log.Debugf("deleting openshift project for broker apps")
+	err := sc.Client.EndUser.CleanupProject(cookie.(string))
+	if err != nil {
+		sc.log.Error(err)
+		errs = append(errs, &TestError{Err: err, Bucket: "DeleteOSBAApp"})
+	}
+	return errs
+}
+
+// CreateOSBAApp creates the OSBA Application under test
+func (sc *SanityChecker) CreateOSBAApp(ctx context.Context) (interface{}, []*TestError) {
+	var errs []*TestError
+	sc.log.Debugf("creating openshift project for broker app")
+	template, err := random.LowerCaseAlphanumericString(5)
```

Comment:
for better or worse, currently the template is hard-coded such that it only works if the namespace is `osba`.  This code can go.